### PR TITLE
feat: add new icons

### PR DESCRIPTION
**Pull request by imrochamatheus**

### CODE REVIEW

File: projects/ion/src/lib/icon/svgs/iconsText/working.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
...
...<path d="M17.7847 2.72204C15.0315 1.40684 11.6325 1.88769 9.34992 4.17025C7.55938 5.96078 6.8778 8.43918 7.30136 10.7515L2.29289 15.76C1.90237 16.1505 1.90237 16.7837 2.29289 17.1742C2.68342 17.5647 3.31658 17.5647 3.70711 17.1742L9.10416 11.7772C9.35658 11.5247 9.45529 11.1569 9.3632 10.812C8.8804 9.00418 9.34899 6.99959 10.7641 5.58446C12.0495 4.29911 13.8219 3.79492 15.4893 4.07448L11.6163 7.94753C11.2258 8.33805 11.2258 8.97121 11.6163 9.36174L14.6381 12.3836C14.8256 12.5711 15.08 12.6765 15.3452 12.6765C15.6104 12.6765 15.8648 12.5711 16.0523 12.3836L19.9254 8.51051C20.2049 10.1779 19.7007 11.9504 18.4154 13.2357C17.0002 14.6508 14.9957 15.1194 13.1878 14.6366C12.8429 14.5445 12.4751 14.6433 12.2227 14.8957L6.82562 20.2927C6.43511 20.6833 6.4351 21.3164 6.82562 21.7069C7.21615 22.0975 7.84931 22.0975 8.23984 21.7069L13.2483 16.6985C15.5607 17.122 18.0391 16.4405 19.8296 14.6499C22.1122 12.3674 22.593 8.96831 21.2778 6.21514C21.1379 5.92223 20.8645 5.71546 20.5445 5.66058C20.2246 5.6057 19.8979 5.70954 19.6684 5.93908L15.3452 10.2622L13.7376 8.65463L18.0608 4.33147C18.2903 4.10193 18.3941 3.77526 18.3393 3.45531C18.2844 3.13537 18.0776 2.86196 17.7847 2.72204Z" fill="#282B33"/>
```
Comment (iurynogueira) [inline projects/ion/src/lib/icon/svgs/iconsText/working.ts:3]: Remove fill attribute

File: projects/ion/src/lib/icon/svgs/iconsText/working.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export default `
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (iurynogueira) [inline projects/ion/src/lib/icon/svgs/iconsText/working.ts:2]: Remove svg tag, the component insert this tag.

**Review by iurynogueira — CHANGES_REQUESTED** [review]
**Nice work!**. some littet fixs.

File: projects/ion/src/lib/icon/svgs/iconsText/working.ts (hunk 1 above)
Comment (iurynogueira) [reply projects/ion/src/lib/icon/svgs/iconsText/working.ts:3]: image

Comment (imrochamatheus) [reply projects/ion/src/lib/icon/svgs/iconsText/working.ts]: Sorry, i removed the others and forgot this one

**Review by iurynogueira — APPROVED** [review]

**Pull request merged**